controllers/capability/capability_configmap.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2024 Broadcom. All Rights Reserved.
+// Broadcom Confidential. The term "Broadcom" refers to Broadcom Inc.
+// and/or its subsidiaries.
+
+package capability
+
+import (
+	"errors"
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// WcpClusterCapabilitiesConfigMapName is the name of the wcp-cluster-capabilities ConfigMap.
+	WcpClusterCapabilitiesConfigMapName = "wcp-cluster-capabilities"
+
+	// WcpClusterCapabilitiesNamespace is the namespace of the wcp-cluster-capabilities
+	// ConfigMap.
+	WcpClusterCapabilitiesNamespace = "kube-system"
+
+	// TKGMultipleCLCapabilityKey is the name of capability key defined in wcp-cluster-capabilities ConfigMap.
+	TKGMultipleCLCapabilityKey = "MultipleCL_For_TKG_Supported"
+)
+
+func NewWcpClusterCapabilitiesConfigMap(wcpClusterConfig map[string]string) (*corev1.ConfigMap, error) {
+	return &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      WcpClusterCapabilitiesConfigMapName,
+			Namespace: WcpClusterCapabilitiesNamespace,
+		},
+		Data: wcpClusterConfig,
+	}, nil
+}
+
+// IsTKGMultipleCLSupported returns if TKG_Multiple_CL is enabled or not in the
```

Comment:
This function is unnecessary. Please just make it a private function in `controllers/capability/capability_configmap_controller.go`.

I will address the use of it in the unit test location down below.